Testscripts/Linux/SetupRDMA.sh

```diff
@@ -118,6 +120,9 @@ function Main() {
 					LogMsg "$?: Installed $req_pkg"
 				;;
 				redhat_8|centos_8)
+					# Due to redhat bug, 1787637, this workaround is required.
+					supplement_pkg="dnf rpm kernel-rpm-macros"
+					install_package $supplement_pkg
```

Comment:
this can be removed, if you use echo 8 > /etc/yum/vars/releasever, if you want to use this, you need put this before install rpm-build

Reply:
Fixed in 8eeb56201567f687899ce7ced0eba6b7d94f3886